File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, space;
    //Get the height of pyramid from user
    do
    {
        height = get_int("Height of pyramid: \n");
    }
    while (height < 1 || height > 8);

    for (int i = 1; i < height + 1; i++)
    {
        //print spaces
        for (space = 0; space < (height - i); space++)
        {
            printf(" ");
        }
        //print left symbol
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        //print 2 spaces in between
        printf("  ");
        //print right symbol
        for (int r = 0; r < i; r++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# Prompt user for height of pyramid
while True:
    try:
        # Only accepts value of 1 to 8
        height = int(input("Height: "))
        if height <= 0 or height >= 9:
            print("Only accepts 1 - 8")
            continue
        break
    except:
        print("Only accepts integer!")
        continue


for x in range(height):
    # Print spaces
    for y in range(height - (x+1)):
        print(" ", end="")

    # Print symbol
    print("#" * (x+1), end="")

    # Print spaces in between
    print(" " * 2, end="")

    # Print symbol
    print("#" * (x+1))
